src/styles/getTooltipStyles.js

```diff
@@ -109,6 +113,11 @@ export default function getTooltipStyles(
             ...tooltipVerticalPadding,
             ...spacing.ph2,
             zIndex: variables.tooltipzIndex,
+            maxWidth: tooltipTextWidth >= maxWidth
```

Comment:
These should be optional right.

Reply:
Yep, they are optional, if maxWidth or tooltipTextWidth are undefined then the ternary operation will return a non valid value which for CSS will mean not setting max-width at all, making tooltip work as currently does.

Reply:
How come? I think it will return false and then a second statement will be run.

Reply:
If `maxWidth` is undefined so will be the`tootipTextWidth`. So when the second statement runs it will try to make this operation: `undefined  + number` which will return `NaN` which is non valid value for CSS. `tootipTextWidth` won't update with recent change in componentDidMount so this operation won't be executed twice. But now I think about it, I will move this logic to a function and make early void return if prop maxWidth is not set. What do you think?

Reply:
My function to set `maxWidth` will return `undefined` or `Number` but according to  [JSDocs style guidelines](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs) it says 'Bad' using `*` on `@returns` on example shown and can't use union types. What should I add on `@returns`?

![image](https://user-images.githubusercontent.com/48106652/162489801-ec1cef13-2178-4ed8-b219-917840e8e707.png)

Reply:
if you notice carefully, we are defining the maxWidth of the object literal. So it should be easy to conditionally add props to the literal value. 

There are many ways to do this. 
1. we create a variable for the returning value. And then add the new props conditionally. 
2. we create a variable whose value is set conditionally to the object of maxWidth, defaults to `{}`. Then we expand this into the return value